### ISSUE TITLE
[Feature Request] - Create 'Setup' Middleware

### ISSUE THREAD

**Issue by SpencerKaiser** [AmericanAirlines/Hangar#298]
### Pre-requisites:
- [x] I looked through both [open and closed issues](../issues?utf8=✓&q=is%3Aissue) and did not find another request for the same or similar feature.

## Describe the solution you'd like

We should add a new middleware handler to check to see if the app has been setup properly. The app should check to make sure the `adminSecret` and `judgeSecret` values have been set and redirect the user to `/setup` if the app has NOT been setup correctly. The handler should continue to check those values until they've been set; once they've been set, the middleware should stop checking the DB in order to redirect if needed.

Dependent on #67

**Comment by SpencerKaiser** [comment]
Closing in favor of #330
